http/e2e/e2e.go

```diff
@@ -0,0 +1,259 @@
+// Copyright 2023 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+// External configurable variables:
+// - CORAZA_ENVOY: Interruptions at response body phase are allowed to return 200 (Instead of 403), but with a body full of null bytes. Defaults to "false".
```

Comment:
I would call it `NULLED_BODY`, I would avoid to make the tests specific to ENVOY.